include/openenclave/internal/sgx/td.h

```diff
@@ -163,6 +163,14 @@ typedef struct _td
     oe_tls_atexit_t* tls_atexit_functions;
     uint64_t num_tls_atexit_functions;
 
+    /* The following information is only available for SGX2 and MISCSELECT is
+     * set to 1 */
+    /* The faulting address associated with the PF exception. Should be zero
+     * for other exception types. */
+    uint64_t faulting_address;
```

Comment:
We need to ensure that these are cleared appropriately so that their values don't linger around.